doc/source/dpf_ansys_file_support_table.rst

```diff
@@ -7,7 +7,7 @@
 +--------------------+------------------------+----------------------------------+----------------------------------+
 |     **Solver**     |    **File format**     |       **Server version**         |       **DPF examples**           |
 +====================+========================+==================================+==================================+
-|                    | .rst, .mode            | **1.0** and later                | :ref:`ref_basic_example`         |
+|                    | .rst, .mode, .rth      | **1.0** and later                | :ref:`ref_basic_example`         |
```

Comment:
@oparreno when would you say RTH started being supported approximately? I would think from the beginning?

Reply:
`mapdl::rth::MeshProvider` has been available since 2020.